app/presenters/content_item_presenter.rb

```diff
@@ -71,6 +71,24 @@ def cache_control_public?
     !content_item.cache_control.private?
   end
 
+  def has_policy_and_engagement_links?
```

Comment:
I think these should probably be moved out into `policy_and_engagement.rb` as it's specific to that. Same with `policy_and_engagement_links_content`